run.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+cd /usr/src/app
+case ${DOCKER_STATE} in
+create)
+    echo "running seed"
+    bundle exec rake db:create
+    bundle exec rake db:migrate
+    bundle exec rake db:seed
```

Comment:
Instead of repeatedly instantiating the whole of Rails, we should be able to speed this all up by saying

```sh
bundle exec rake db:create db:schema:load db:seed
```

The same goes for other sections.